MapboxNavigation/FeedbackItem.swift

```diff
@@ -16,71 +16,71 @@ public extension FeedbackType {
         case .general:
             return "Feedback"
         case .incorrectVisual(.none):
-            return "Incorrect visual"
+            return NSLocalizedString("INCORRECT_VISUAL_FEEDBACK", bundle: .mapboxNavigation, value: "Incorrect visual", comment: "General category of route feedback where visual instruction was incorrect.")
```

Comment:
Can you make the same change to ”Looks Incorrect” as in https://github.com/mapbox/mapbox-navigation-ios/pull/2497#discussion_r462691238?

Reply:
Done.